studdy-buddy/src/app/footer/footer.component.css

```diff
@@ -6,31 +6,20 @@
     margin: 0;
     padding: 0;
     box-sizing: border-box;
- 
+    z-index: 1200;
+    white-space: nowrap;
   }
-    
+
   a {
     color: #000000;
     text-decoration: none;
   }
-  
-  a:hover {
```

Comment:
Instead of making links larger on hovering, underlining is more suitable for text I think